src/ScheduledTasks/Client/ScheduledTaskClientImpl.cs

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DurableTask.ScheduledTasks;
+
+/// <summary>
+/// Client for managing scheduled tasks in a Durable Task application.
+/// </summary>
+#pragma warning disable CA1711 // Identifiers should not have incorrect suffix
+public class ScheduledTaskClientImpl(DurableTaskClient durableTaskClient, ILogger logger) : ScheduledTaskClient
+#pragma warning restore CA1711 // Identifiers should not have incorrect suffix
+{
+    readonly DurableTaskClient durableTaskClient = Check.NotNull(durableTaskClient, nameof(durableTaskClient));
+    readonly ILogger logger = Check.NotNull(logger, nameof(logger));
+
+    /// <inheritdoc/>
+    public override async Task<ScheduleClient> CreateScheduleAsync(ScheduleCreationOptions creationOptions, CancellationToken cancellation = default)
```

Comment:
should not duplicate implementation code between ScheduleClientImpl.cs and ScheduledTaskClientImpl.cs